test/data/confirmations/contract-interaction.ts

```diff
@@ -160,4 +164,13 @@ export const genUnapprovedContractInteractionConfirmation = ({
     userEditedGasLimit: false,
     userFeeLevel: 'medium',
     verifiedOnBlockchain: false,
-  } as SignatureRequestType);
+  } as SignatureRequestType;
+
+  // Overwrite simulation data if provided
+  if (simulationData) {
+    console.log('>>>>>', simulationData);
```

Comment:
I guess this is forgotten

Reply:
removed https://github.com/MetaMask/metamask-extension/pull/27594/commits/1aec7d843977cf3324937f1377891a499af9fff0